.github/actions/slack-report/action.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Report Workflow Status to Slack"
+description: "Report the workflow status to Slack to help identify breakages faster."
+inputs:
+  owner:
+    description: "The Slack ID of the person to be tagged."
+    default: "U014XCQ9CF8" # @Raymond Kim
+  slack_webhook_url: 
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Report Github Pipeline Status Slack Action 
+      uses: slackapi/slack-github-action@v1.26.0
+      with:
+        payload: |
+          {
+            "text": "just so you know `${{ github.event.sender.login }}` broke ${{ github.workflow }} with https://github.com/tenstorrent/tt-metal/commit/${{ github.sha }}",
```

Comment:
It would be better if the `${{ github.event.sender.login }}` was part of github link as they are not being highlighted as an object or figure out a way of how to tag a person in Slack.